test/index-server.test.ts

```diff
@@ -107,16 +106,18 @@ describe("Client", () => {
     );
   });
 
-  test("works with non-JSON erroring response", async () => {
+  // NOTE(bhinchley): not sure this test is needed, not sure why the api
+  // would return a non json response.
+  test.skip("works with non-JSON erroring response", async () => {
```

Comment:
I think you've handled this so we can un-skip it now?

In general I've seen non-JSON responses come back when load balancers fall over and things of that ilk.